components/feature/toolbar/src/main/java/mozilla/components/feature/toolbar/ToolbarPresenter.kt

```diff
@@ -67,9 +67,10 @@ class ToolbarPresenter(
         toolbar.setSearchTerms(session.searchTerms)
     }
 
-    override fun onLoadingStateChanged() { }
-
-    override fun onNavigationStateChanged() { /* TODO */ }
-
-    override fun onSecurityChanged() { /* TODO */ }
+    override fun onCustomTabConfigChanged() {
+        session.customTabConfig?.toolbarColor?.let {
+            toolbar.setBackgroundColor(it)
+        }
+        // TODO Apply remaining configurations: https://github.com/mozilla-mobile/android-components/issues/306
```

Comment:
I wonder if this should be done in the callback: Afaik this only works if the callback is set before the custom tab config is set, right? Otherwise the callback won't get executed, or? Especially as the config will never change I wonder if we should just do this on session selection / initializeView?